src/main/scala/it/cnr/isti/hpc/wikipedia/parallel/ParallelJsonpediaParser.scala

```diff
@@ -8,16 +8,27 @@ import org.apache.commons.io.FileUtils
 
 object ParallelJsonpediaParser{
 
+  def getHeader(pathToSingleWikiDump: String): String ={
+    // First line contains the header:
+    // <mediawiki xmlns="http://www.mediawiki.org/xml/export-0.10/"....
+    scala.io.Source.fromFile(pathToSingleWikiDump).getLines().next()
+  }
+
+  def getNamespaces(pathToSingleWikiDump: String): String ={
+    val firstLines = scala.io.Source.fromFile(pathToSingleWikiDump).getLines().slice(0, 90000).toList
```

Comment:
actually, just got the answer I wanted: 
http://stackoverflow.com/questions/34316979/wikipedia-xml-dump-where-to-get-translations-for-disambiguation-directives